scientific_library/tvb/simulator/monitors.py

```diff
@@ -261,7 +261,9 @@ def config_for_sim(self, simulator):
         # setup given spatial mask or default to region mapping
         if self.spatial_mask is None:
             self.is_default_special_mask = True
-            if not (simulator.surface is None):
+            if simulator.surface is not None and simulator.connectivity is not None:
```

Comment:
I think 'simulator.conn' is always not None.

Reply:
I would say so too, thanks